docs/understand/performance-best-practices.md

```diff
@@ -31,9 +26,61 @@ When accessing data using the branch name (e.g. `lakefs://repo/main/path`) lakeF
 For more information, see [how uncommitted data is managed in lakeFS][representing-refs-and-uncommitted-metadata].
 
 ## Operate directly on the storage
```

Comment:
If we want to breakdown this section down into sub-sections, i'd suggest that each section describe a way to operate directly on the storage rather than distinguishing between reads and writes that are supported by most ways. That is, I'd use a structure like:

## Operate directly on the storage
### Pre-sign URLs 
### lakeFS Hadoop Filesystem 
### Staging API  

WDYT?

Reply:
I created a pr with the suggestions